README.md

```diff
@@ -121,7 +121,7 @@ By default, the remote tests, which run against the real backend server, are
 skipped. To enable them, set the following environment variables:
 
 ```shell
-export PYTKET_RUN_REMOTE_TESTS=true
+export PYTKET_RUN_REMOTE_TESTS=1
```

Comment:
Python only checks of this variable it set, not depended of the value.

Reply:
I know, I just wanted to pedantically make the value the same as in the CI :-)